sql-parser/src/main/java/io/crate/sql/tree/Window.java

```diff
@@ -49,6 +60,49 @@ public Window(List<Expression> partitions, List<SortItem> orderBy, Optional<Wind
         return windowFrame;
     }
 
+    /**
+     * Merges the provided window definition into the current one
+     * by following the next merge rules:
+     * <ul>
+     * <li> The provided window cannot override partition or order by clauses.
+     * <li> The provided window cannot define the window frame.
```

Comment:
It does not work if you run it on a table instead of `generated_series`. The pg documentation of the `window` clause says `The new window always uses its own frame clause; the copied window must not specify a frame clause.`, see https://www.postgresql.org/docs/11/sql-select.html. But i am not sure what is the reasoning there.